timelock-server/src/suiteTest/java/com/palantir/atlasdb/timelock/MultiNodePaxosTimeLockServerIntegrationTest.java

```diff
@@ -78,6 +78,15 @@ public void nonLeadersReturn503() {
         });
     }
 
+
+    @Test
+    public void nonLeadersReturn503_conjure() {
+        cluster.nonLeaders(client.namespace()).forEach((namespace, server) -> {
```

Comment:
this test verifies that the code works in jersey... but we care more about undertow. Witchcraft is not open sourced so this test is hard to write. When we merge this, I'm going to write a similar test in our internal dist of timelock.